jlm/hls/backend/rvsdg2rhls/instrument-ref.cpp

```diff
@@ -90,10 +90,10 @@ instrument_ref(llvm::RvsdgModule & rm)
   //  addr, width, memstate
   jlm::llvm::FunctionType loadFunctionType(
       { jlm::llvm::PointerType::Create().get(),
-        &jlm::rvsdg::bit64,
-        llvm::iostatetype::create().get(),
+        &*jlm::rvsdg::bittype::Create(64),
```

Comment:
Is this not equivalent to `jlm::rvsdg::bittype::Create(64).get()`?

The pattern is all over the place. I will only comment on it here.

Reply:
Yes that is correct, and it is (mostly) to specific to functiontype which wants pointers. The "&*" pattern will go way when this is converted to also used shared_ptr -- and then "&*" is a bit easier to search&replace for than "something.get()"

Reply:
Got it. Thanks!